roles/mirror_ocp_release/tasks/facts.yml

```diff
@@ -7,12 +7,27 @@
   register: release_image
   changed_when: false
 
-- name: "Write rhcos.json from command (>= 4.8)"
+- name: Write rhcos.json from command (4.16+)
+  when: mor_version is version("4.16", ">=")
+  containers.podman.podman_container:
+    name: extract-rhcos
+    image: quay.io/centos/centos:stream9
+    command:
+      - /bin/bash
+      - -c
+      - "/ocp/openshift-baremetal-install coreos print-stream-json > /ocp/rhcos-test.json"
+    rm: true
```

Comment:
I guess there is no need of these options right?
```
        interactive: no
        tty: yes
        detach: yes
        debug: true
```

Reply:
Correct, those are not required